jsapp/js/routerUtils.ts

```diff
@@ -43,111 +43,107 @@ export function isNewLibraryItemRoute() {
   return getCurrentPath() === ROUTES.NEW_LIBRARY_ITEM;
```

Comment:
This is going to cause conflicts with #3342, I think we should merge that in first, then do the proper typescript conversion for those changes, then merge this